lib/Configuration.js

```diff
@@ -23,6 +23,19 @@ class Configuration {
 
             try {
                 this.settings = Object.assign(this.settings, JSON.parse(fs.readFileSync(this.location, {"encoding": "utf-8"})));
+                // Migrate legacy "areas" to "zones"
```

Comment:
Should this be in the Configuration.js or would it be better to have it somewhere else? This should be a viomi function? As such I think it should be somewhere else. (Just my opinion, feel free to disagree and ignore)

Reply:
This is to switch users of older Valetudo versions seamlessly to the new format. Maybe worth to extract a separate method though.

Reply:
Done.